src/Cleaners.py

```diff
@@ -0,0 +1,80 @@
+import re, pickle, os
```

Comment:
Imports should be all on a separate line. See for example the PEP-8 guide: 

![image](https://user-images.githubusercontent.com/58271113/171017526-bbae7e84-be12-4959-855f-af3dcad6ff5c.png)